src/App.jsx

```diff
@@ -11,7 +11,6 @@ export default function App() {
   const [gender, setGender] = useState("");
   const [submitForm, setSubmitForm] = useState([]);
```

Comment:
it is an array and the name of state is submit form, does not makes any sense. Please try to use appropriate names for state variables

Reply:
Resolved